### PR TITLE
Fix grpcPort and httpPort in v1 CRD

### LINKED ISSUE #2866: V1 CRD has missing grpcPort and httpPort

### PR DESCRIPTION
**Pull request by cliveseldon**
**What this PR does / why we need it**:

 * Fixes the V1 CRD (k8s >= 1.18) which was missing `grpcPort` and `httpPort` in OpenAPISchema in CRD definition.
 * Fixes protocol_examples notebook which had old `grpc_port` in example

**Which issue(s) this PR fixes**:

Fixes #2866

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
```

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **cliveseldon**
You can assign the PR to them by writing `/assign @cliveseldon` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Fri Jan 22 10:29:33 UTC 2021
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2868/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2868 --build=2

**Comment by seldondev** [comment]
Fri Jan 22 10:29:38 UTC 2021
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2868/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2868 --build=1

File: helm-charts/seldon-core-operator/templates/webhook.yaml [hunk 1]
```diff
@@ -4,22 +4,8 @@
 {{- $cert := genSignedCert "seldon-webhook-service" nil $altNames 365 $ca -}}
 ---
 
-{{- if not .Values.certManager.enabled -}}
-apiVersion: v1
```
Comment (axsaucedo) [inline helm-charts/seldon-core-operator/templates/webhook.yaml:8]: Why is this being removed as part of the PR?

Comment (axsaucedo) [reply helm-charts/seldon-core-operator/templates/webhook.yaml:8]: Oh I see, git just sees the diff as it was moved

**Pull request merged**